packages/gitbook/src/components/Header/Dropdown.tsx

```diff
@@ -111,31 +111,38 @@ export function DropdownMenu(props: { children: React.ReactNode }) {
  * Menu item in a dropdown.
  */
 export function DropdownMenuItem(props: {
-    href: string;
+    href: string | null;
     active?: boolean;
+    className?: ClassValue;
```

Comment:
I'm not familiar with the GBO dev env yet : why isn't that a string?

Reply:
We manipulate `clsx` values, but IMO we should not. All components should only accept a `className?: string` and if you need to merge thing you can do it when you pass the prop: `<Any className={clsx(...)} />` like we do in the app.